.github/workflows/build.yml

```diff
@@ -34,9 +34,9 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install and build
+    - name: yarn install and build
       run: |
-        npm ci
+        rm -rf node_modules && yarn install --frozen-lockfile
```

Comment:
I am not sure we expect this. Wait for @Fine0830

Reply:
thank you , i change to wepack . can you see the last file commit?

Reply:
I am not familiar with frontend codes. Will wait for the reviewer.

Reply:
can you open the issue entrance? i have some question to ask.
example:
when I use the node_module, it show the error message.
import './lib/index';
^^^^^^
SyntaxError: Cannot use import statement outside a module

Reply:
You could discuss here if it is relative to your PR.